.copywrite.hcl

```diff
@@ -0,0 +1,16 @@
+schema_version = 1
+
+project {
+  license        = "MPL-2.0"
+  copyright_year = 2021
```

Comment:
Technically this should be 2020 since this repo was created in 2020, though in practice having the wrong year here doesn't really have any consequences.